torch/ao/quantization/__init__.py

```diff
@@ -141,17 +146,21 @@
 ]
 
 def default_eval_fn(model, calib_data):
-    r"""
+    r"""Define the default evaluation function.
+
     Default evaluation function takes a torch.utils.data.Dataset or a list of
     input Tensors and run the model on the dataset
     """
     for data, target in calib_data:
         model(data)
 
 class _DerivedObserverOrFakeQuantize(ObserverBase):
-    r""" This observer is used to describe an observer whose quantization parameters
+    r"""Define an observer whose parameters are defined from other observers.
```

Comment:
This is a private class, and as such does not need to adhere to the same strict rules as others